src/plugins/home/public/plugin.ts

```diff
@@ -166,6 +166,9 @@ export class HomePublicPlugin
       navLinkStatus: AppNavLinkStatus.hidden,
       mount: async (params: AppMountParameters) => {
         const [coreStart, { navigation }] = await core.getStartServices();
+        if (!!coreStart.application.capabilities.workspaces?.enabled) {
+          coreStart.application.navigateToApp('workspace_initial');
```

Comment:
```suggestion
          coreStart.application.navigateToApp('workspace_initial');
          return ;
```
What about we return early here to avoid unnecessary code execution below.

Reply:
Sure, I will update it.